scripts/gen_jvt.py

```diff
@@ -244,22 +272,24 @@ def remove_r1_from_path(path):
         default=2 * multiprocessing.cpu_count(),
     )
     args = parser.parse_args()
+
     generator = JVTGenerator(
-        "AVCv1",
-        "JVT-AVC_V1",
-        Codec.H264,
-        "JVT AVC version 1",
-        H264_URL
+        "AVCv1", "JVT-AVC_V1", Codec.H264, "JVT AVC version 1", H264_URL
     )
     generator.generate(not args.skip_download, args.jobs)
 
     generator = JVTGenerator(
-        "SVC",
-        "JVT-SVC_V1",
+        "MVC",
+        "JVT-MVC",
         Codec.H264,
-        "JVT SVC version 1",
+        "JVT Multiview Video Coding test suite",
         H264_URL,
-        True
+        True,
+    )
+    generator.generate(not args.skip_download, args.jobs)
+
+    generator = JVTGenerator(
+        "SVC", "JVT-SVC_V1", Codec.H264, "JVT SVC version 1", H264_URL, True
```

Comment:
Only a minor change, I don't understand why one enum is separated by line breaks and others are not...

Reply:
It's a matter of automatic syntax correction that black formatter does.
I will try to unify